Description: Fix unicode string in Python 3.2
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-08-08

--- oslo-config-1.4.0.0~a3.orig/oslo/config/generator.py
+++ oslo-config-1.4.0.0~a3/oslo/config/generator.py
@@ -107,6 +107,7 @@ import logging
 import operator
 import sys
 import textwrap
+import six
 
 from oslo.config import cfg
 import stevedore.named  # noqa
@@ -170,7 +171,7 @@ class _OptFormatter(object):
 
         opt_type = self._TYPE_DESCRIPTIONS.get(type(opt), 'unknown type')
 
-        help_text = u'%s(%s)' % (opt.help + ' ' if opt.help else '', opt_type)
+        help_text = six.u('%s(%s)') % (opt.help + ' ' if opt.help else '', opt_type)
         if self.wrap_width is not None and self.wrap_width > 0:
             lines = [textwrap.fill(help_text,
                                    self.wrap_width,
